.gitignore

```diff
@@ -1 +1,4 @@
 brooklyn-cli
+.idea
+*.iml
+temp_test.go
```

Comment:
Note - the addition of temp_test.go is deliberate.   This is to allow one to have the file in question there for quick test purposes.